MonoGame.Framework/Audio/SoundEffect.cs

```diff
@@ -262,13 +262,7 @@ public static int GetSampleSizeInBytes(TimeSpan duration, int sampleRate, AudioC
         /// </remarks>
         public bool Play()
         {
-            var inst = GetPooledInstance(false);
-            if (inst == null)
-                return false;
-
-            inst.Play();
-
-            return true;
+            return Play(1f, 0f, 0f);

```

Comment:
Currently the `Play(float volume, float pitch, float pan)` sets these twice... once to the default and once to the actual desired values.  `Play()` avoids this double setting currently.

So if we wanted to improve things here I would like to see the volume/pitch/pan be passed down into the pool so it can reset the values one time to the right state.